gpt/commit.js

```diff
@@ -252,6 +252,61 @@ const runGenerativeCommit = async () => {
     }
```

Comment:
The tools are hard-coded in the function, which reduces the modularity of the code. Consider passing them as parameters to the function.